Mobile/starter_project_matador/lib/features/example/presentation/screen/signup.dart

```diff
@@ -0,0 +1,216 @@
+// Import flutter heplper library
+import 'package:flutter/material.dart';
+
+class SignUp extends StatelessWidget {
```

Comment:
Changing the name to SignUpPage would be more descriptive because there will be other identifiers called signup when we implement logic.